lib/components/artist/shows.js

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { View, Text, StyleSheet, Image } = React;
+
+import Headline from '../text/headline';
+import SerifText from '../text/serif'
+
+export default class Shows extends React.Component {
+	
+	render() {
+	  return (
+	  	<View style={styles.container}>
+		  	<SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+				<Image style={styles.thumbnail} />
+				<Text style={styles.sansSerifText}> {this.props.show.partner_name.toUpperCase()} </Text>
+				<Text style={styles.sansSerifText}> {this.showTypeString(this.props.show)} </Text>
+				<SerifText style={{marginTop: 2}}> {this.props.show.title} </SerifText>
+				<SerifText style={{color: 'grey', marginTop: 2}}> {this.dateAndLocationString(this.props.show)} </SerifText>
+			</View>
+		);
+	}
+	
+	showTypeString(show) {
+		return show.type.toUpperCase() + ', ' + show.number_of_works + ' WORKS'
```

Comment:
The indents in VS are set to 2...  🤔

![screen shot 2016-04-21 at 5 42 44 pm](https://cloud.githubusercontent.com/assets/2712962/14725192/7dc0bcfa-07e8-11e6-9f25-a7e594f8f431.png)